go.mod

```diff
@@ -5,20 +5,15 @@ go 1.16
 
 require (
 	github.com/StackExchange/wmi v0.0.0-20210224194228-fe8f1750fd46 // indirect
-	github.com/andybalholm/brotli v1.0.2 // indirect
 	github.com/go-ole/go-ole v1.2.5 // indirect
-	github.com/klauspost/compress v1.12.2 // indirect
-	github.com/klauspost/pgzip v1.2.5 // indirect
 	github.com/mattn/go-runewidth v0.0.12 // indirect
-	github.com/mholt/archiver/v3 v3.5.0
```

Comment:
now that e2e is split, there's no reference here anymore nor any indirect stuff in go.sum